package io.openfuture.api.exception

/**
 * @author Kadach Alexey
 */
class FunctionCallException(message: String?) : RuntimeException(message)